src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using `eslint-disable` for `@typescript-eslint/no-explicit-any`. Instead, try to avoid `any` type. It's better to use a more specific type. You can define a type or interface for the data.